app/src/main/res/layout/onboarding_fragment.xml

```diff
@@ -43,62 +43,61 @@
       <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/get_started_container"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:paddingStart="56dp"
-        android:paddingEnd="56dp"
-        android:paddingBottom="24dp"
-        android:visibility="@{viewModel.slideNumber == (viewModel.totalNumberOfSlides - 1) ? View.VISIBLE: View.GONE, default=gone}">
+        android:layout_height="wrap_content">
 
-        <Button
-          android:id="@+id/get_started_button"
+        <View
+          android:id="@+id/onboarding_fragment_divider_view"
           android:layout_width="match_parent"
-          android:layout_height="wrap_content"
-          android:background="@drawable/primary_rounded_button"
-          android:fontFamily="sans-serif-medium"
-          android:minHeight="48dp"
-          android:onClick="@{(v) -> presenter.clickOnGetStarted()}"
-          android:text="@string/get_started"
-          android:textColor="@color/white"
-          android:textSize="14sp"
-          app:layout_constraintBottom_toBottomOf="parent"
-          app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
-      </androidx.constraintlayout.widget.ConstraintLayout>
-
-      <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/bottom_container"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:paddingStart="28dp"
-        android:paddingEnd="28dp"
-        android:paddingBottom="16dp"
-        android:visibility="@{viewModel.slideNumber != (viewModel.totalNumberOfSlides - 1) ? View.VISIBLE: View.GONE, default=visible}">
+          android:layout_height="1dp"
+          android:layout_marginBottom="16dp"
+          android:background="@color/oppiaPrimaryText"
+          app:layout_constraintBottom_toTopOf="@+id/slide_dots_container" />
 
         <LinearLayout
           android:id="@+id/slide_dots_container"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
+          android:layout_marginTop="32dp"
+          android:layout_marginBottom="16dp"
+          android:minHeight="48dp"
+          android:gravity="center"
           android:orientation="horizontal"
           app:layout_constraintBottom_toBottomOf="parent"
           app:layout_constraintEnd_toEndOf="parent"
-          app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent" />
+          app:layout_constraintStart_toStartOf="parent" />
+
+        <ImageView
+          android:id="@+id/onboarding_fragment_next_image_view"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginEnd="32dp"
```

Comment:
The margin that you are checking is from the image but you should check the margin from the `box` around the image which is `20dp`
<img width="131" alt="Screenshot 2020-04-28 at 11 38 31 AM" src="https://user-images.githubusercontent.com/9396084/80452873-cb4faf80-8944-11ea-94bb-f1524125905f.png">


Also, marginTop and bottom is `12dp` which you can also add as `12dp` padding top and bottom for its parent layout.

Reply:
> The margin that you are checking is from the image but you should check the margin from the `box` around the image which is `20dp`

Done
><img alt="Screenshot 2020-04-28 at 11 38 31 AM" width="131" src="https://user-images.githubusercontent.com/9396084/80452873-cb4faf80-8944-11ea-94bb-f1524125905f.png">
> 
> Also, marginTop and bottom is `12dp` which you can also add as `12dp` padding top and bottom for its parent layout.

There is no parent as such as displayed in mock as we are using ConstraintLayout ... they are aligned as per the alignment with nearby siblings in mock.